src/components/JitsiMeeting.tsx

```diff
@@ -0,0 +1,90 @@
+import { useEffect, useRef, useState } from "react";
```

Comment:
Please use our eslint base: https://github.com/jitsi/eslint-config-jitsi which has the main rules we follow. Single quotes being one of them ;-)